src/components/Main/Main.js

```diff
@@ -11,20 +11,32 @@ export default function Main() {
       uniqueGenreArray.push(movie.genre.split("|")[0]);
     }
   });
+  // {Genre1: [{movie1}, {movie2},
+  // Genre2: [{movie1}{movie2}{movie3}]
+  // ]}
   let sortedGenreMovies = {};
   uniqueGenreArray.forEach((uniqueGenre) => {
     let alikeGenreMovies = [];
     movieData.forEach((movie) => {
       if (movie.genre === uniqueGenre) {
-        alikeGenreMovies.push(movie.title);
+        alikeGenreMovies.push(movie);
         sortedGenreMovies[uniqueGenre] = alikeGenreMovies;
       }
     });
   });
-  console.log(sortedGenreMovies);
   return (
-    <div className="center">
-      <Movie />
+    <div className="main">
+      {Object.keys(sortedGenreMovies).map((genre) => (
+        <div key={genre}>
+          {/* Each genre */}
+          <h1>{genre}</h1>
+          {console.log(Object.values(sortedGenreMovies[genre])[0].image)}
+          <div>{Object.values(sortedGenreMovies[genre].map((movie) => (
+            <img className="movie--icon" src={movie.image}></img>
+          )))}</div>
```

Comment:
I think something is a bit off here
```suggestion
          <div>{Object.values(sortedGenreMovies[genre]).map((movie) => (
            <img className="movie--icon" src={movie.image}></img>
          ))}</div>
```

Reply:
Also, we should make a Movie component and use that instead of the  `img` I know for now its just an img, but this will change when we add routing and other features. Also, dont forget a key

```suggestion
            <div>{Object.values(sortedGenreMovies[genre]).map((movie) => (
            <Movie key={movie.id} movie={movie} />
          ))}</div>
```